buildSrc/src/main/java/org/opensearch/gradle/testclusters/OpenSearchNode.java

```diff
@@ -341,6 +345,49 @@ public void module(String moduleProjectPath) {
         module(maybeCreatePluginOrModuleDependency(moduleProjectPath));
     }
 
+    @Override
+    public void extension(ExtensionsProperties extensions) {
+        this.extensions.add(extensions);
+    }
+
+    public void writeExtensionFiles() {
+        try {
+            // Creates extensions.yml in the target directory
```

Comment:
Can we create a new/empty extensions.yml file rather creating it manually. The same way we create opensearch.yml?
You can use `new File()` here

Reply:
Isn't that constructor/Object on the forbidden API list?  I'm a little confused on the ask here.  Why do we want an empty extensions.yml file?

Reply:
Created an issue to address this in the future: https://github.com/opensearch-project/OpenSearch/issues/7368